app/views/user_mailer/challenge_assignment_notification.html.erb

```diff
@@ -49,20 +47,18 @@
   <%= styled_divider %>
 
   <p>
-    <%= style_bold("This assignment is due at:") %> <%= time_in_zone(@collection.challenge.assignments_due_at, (@collection.challenge.time_zone || Time.zone.name), @assigned_user) %>.
+    <%= style_bold(t(".due")) %> <%= time_in_zone(@collection.challenge.assignments_due_at, (@collection.challenge.time_zone || Time.zone.name), @assigned_user) %>.
   </p>
 
-  <p>
-    You can look up this assignment from <%= style_link("your Assignments page", user_assignments_url(@assigned_user)) %>.
-  </p>
+  <p><%= t ".html.look_up", link: style_link(t(".html.look_up_link"), user_assignments_url(@assigned_user)) %></p>
```

Comment:
This also appears to be giving exposed HTML: `You can look up this assignment from &lt;a href=&quot;http://archiveofourown.org/users/myname4/assignments&quot; style=&quot;color:#990000&quot;&gt;your Assignments page&lt;/a&gt;.`